tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -54,15 +54,28 @@
         width:30%;
       }
       .right-pane {
-        width:70%;
+        width:100%;
         height:300px;
       }
       .table-style {
         table-layout:fixed;
         width:95%;
       }
       .performance-table {
-        width: 100%;
+        width:100%;
+      }
+      .x-axis-title {
```

Comment:
For both `x-axis-title` and `y-axis-title`, can we specify `text-transform: capitalize;`? That looks a bit more polished IMO. :)

Reply:
done. thanks for the tips